bin/Notebooks/PyRadiomics Example.ipynb

```diff
@@ -1,481 +0,0 @@
-{
```

Comment:
@blezek, you deleted this example, was there a specific reason for this? This example was also used in the short video we made on PyRadiomics usage, and was therefore also made available in the repository